homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
While I hate that I have to agree with this as I have a few automations tied to this, but you're both obviously right...